components/brave_rewards/resources/page/components/pageWallet.tsx

```diff
@@ -243,6 +256,17 @@ class PageWallet extends React.Component<Props, State> {
       }
     }
 
+    const { externalWallet } = this.props.rewardsData
+    if (externalWallet && disconnectWalletError) {
+      return {
+        node: <><b>{getLocale('uhOh')}</b><br />{getLocale('disconnectWalletFailed').replace('$1', this.toCustodian(externalWallet.type))}</>,
```

Comment:
There's a function `utils.getWalletProviderName(externalWallet)` that you can use here to get the display-friendly wallet provider name.

Reply:
updated 👍